### PR TITLE
remove microsite from build for now

### PR DESCRIPTION
**Pull request by natansil**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -311,11 +311,11 @@ workflows:
...
-          requires:
-            - microsite
-          filters:
-            <<: *filter_tags
-            branches:
-              ignore: /.*/
+      # - microsite:
+      #     context: Website
+      #     requires:
+      #       - microsite
```
Comment (mijicd) [inline .circleci/config.yml:317]: This one even had a circular dependency, so make sure to fix that before eventually uncommenting :).

**Review by mijicd — COMMENTED** [review]
You can probably drop the mdoc dependency for now, and later on we'll introduce all of them. Other than that, it looks good.

**Comment by natansil** [comment]
@mijicd done.

File: .circleci/config.yml (hunk 1 above)
Comment (natansil) [reply .circleci/config.yml:317]: right :)

**Review by mijicd — APPROVED** [review]

**Pull request merged**